test/screens/v2/widget_instance/reconstructed_alert_test.exs

```diff
@@ -378,23 +485,19 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlertTest do
         widget
         |> put_effect(:shuttle)
         |> put_informed_entities([
-          ie(stop: "place-dwnxg", route: "Red"),
-          ie(stop: "place-dwnxg", route: "Orange")
+          ie(stop: "place-mlmnl", route: "Orange", route_type: 1)
         ])
         |> put_cause(:unknown)
+        |> put_is_full_screen(true)
 
       expected = %{
-        issue: %{icon: nil, text: ["No", %{route: "red"}, %{route: "orange"}, "trains"]},
-        location: "at Downtown Crossing",
-        cause: "",
-        routes: [
-          %{color: :orange, text: "OL", type: :text},
-          %{color: :red, text: "RL", type: :text}
-        ],
+        issue: "No trains",
+        location: "Shuttle buses replace Orange Line trains at Malden Center",
+        cause: nil,
         effect: :shuttle,
-        urgent: true,
-        region: :inside,
```

Comment:
When merge conflicts are resolved between this and the backend branch, I think `region` will need to be added back in, because it's used by the frontend.

Reply:
Similarly, `endpoints` was added, and so was `unaffected_routes` in certain cases